charts/victoria-metrics-anomaly/values.yaml

```
@@ -126,27 +126,10 @@ config:
   preset: ""
   # -- [Scheduler section](https://docs.victoriametrics.com/anomaly-detection/components/scheduler/)
   schedulers:
```

Let's set it to an empty map to make it explicit.
```suggestion
  schedulers: {}
```